code/modules/clothing/head/misc_special.dm

```diff
@@ -272,11 +272,28 @@
 
 /obj/item/clothing/head/wig/afterattack(mob/living/carbon/human/target, mob/user)
 	. = ..()
-	if (istype(target) && (HAIR in target.dna.species.species_traits) && target.hairstyle != "Bald")
-		to_chat(user, "<span class='notice'>You adjust the [src] to look just like [target.name]'s [target.hairstyle].</span>")
-		add_atom_colour("#[target.hair_color]", FIXED_COLOUR_PRIORITY)
-		hairstyle = target.hairstyle
-		update_appearance()
+	if(istype(target))
```

Comment:
do an early return here, indentation bad, you know the deal

Reply:
Done 👍